src/main/resources/jupyter/init-actions.sh

```diff
@@ -323,6 +324,15 @@ if [[ "${ROLE}" == 'Master' ]]; then
         docker cp /etc/${JUPYTER_SAFE_MODE_PLUGIN} ${JUPYTER_SERVER_NAME}:${JUPYTER_USER_HOME}/.jupyter/custom/
       fi
 
+      # If edit-mode.js was specified, copy it into the jupyter docker container.
+        if [ ! -z ${JUPYTER_EDIT_MODE_PLUGIN_URI} ] ; then
+          log 'Installing edit-mode.js extension...'
+          gsutil cp ${JUPYTER_EDIT_MODE_PLUGIN_URI} /etc
+          JUPYTER_EDIT_MODE_PLUGIN=`basename ${JUPYTER_EDIT_MODE_PLUGIN_URI}`
+          retry 3 docker exec ${JUPYTER_SERVER_NAME} mkdir -p ${JUPYTER_USER_HOME}/.jupyter/custom
```

Comment:
Just curious, is there a reason to use `~/.jupyter/custom` instead of installing this as a "normal" extension (see the block starting on L244)?

Also I'm wondering if this works out of the box alongside `google_sign_in.js` (which is also in `~/jupyter/custom`). Am wondering if this needs updating to reference `edit-mode.js` and `safe-mode.js`: https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/google_plugin_jupyter.js